op-e2e/system/altda/concurrent_test.go

```diff
@@ -43,7 +43,7 @@ func TestBatcherConcurrentAltDARequests(t *testing.T) {
 	l2Seq := sys.NodeClient("sequencer")
 
 	// we wait for numL1TxsExpected L2 blocks to have been produced, just to make sure the sequencer is working properly
-	_, err = geth.WaitForBlock(big.NewInt(numL1TxsExpected), l2Seq, time.Duration(cfg.DeployConfig.L2BlockTime*uint64(numL1TxsExpected))*time.Second)
+	_, err = geth.WaitForBlock(big.NewInt(numL1TxsExpected), l2Seq)
```

Comment:
(comment for all of these test changes) -- looks like all the tests are losing their configured value in favor of the defaults -- do you think it would be worth it to retain the set-values as `waitForBlockOptions`? Presumably the values we're deleting here would map to the `absoluteTimeout`, and then tests more prone to error could get shorter `noChangeTimeout` values?

Reply:
I did this on purpose. I wanted to set a sensible default value for the absolute timeout that takes into account how the tests behave when the executor is under load. This kind of timeout math only works when load is minimal.

The default timeout is quite long, and will hopefully reduce the number of deadline flakes we see.